README.md

```diff
@@ -42,9 +42,18 @@ defined [here](https://github.com/icon-project/IIPs/blob/master/IIPS/iip-52.md)
     - [Installation  ](#installation--)
     - [Getting Started  ](#getting-started--)
         - [Prerequisite](#prerequiste)
+        - [Build](#build)
         - [Running the tests](#testing)
+        - [Deploy](#deploy)
+    - [Developing IBC Dapp  ](#developing-ibc-dapp--)
+    - [Contributing  ](#contributing--)
+    - [License  ](#license--)
+    - [Contact  ](#contact--)
 
 ## About <a name = "about"> </a>
+Relayer for these contracts deviates slightly from official cosmos relayer due to the fact that ICON uses BTP Blocks as storage for messages and also on cosmwasm side the ibc host is deployed as contracts rather than the native ibc host module of cosmos chain. Relayer for ibc-icon can be found [here](https://github.com/icon-project). 
```

Comment:
```suggestion
Relayer for these contracts deviates slightly from official cosmos relayer due to the fact that ICON uses BTP Blocks as storage for messages and also on cosmwasm side the ibc host is deployed as contracts rather than the native ibc host module of cosmos chain. Relayer for ibc-icon can be found [here](https://github.com/icon-project/ibc-relay). 
```